PROCESSOUT-247: update the README file (#64)
processout/documentation: README.md

# Documentation

This is the Ruby version of the documentation.

> It is based on [Slate](https://github.com/slatedocs/slate) and helps you create beautiful, intelligent, responsive API documentation.

Reviewer: ```suggestion
This is the documentation website for [ProcessOut](https://www.processout.com) (https://docs.processout.com). It is based on [Slate](https://github.com/slatedocs/slate).
```

`This is the Ruby version of the documentation.` was misleading because it implies there are other versions. My suggested change has more relevant information about the repo and a shortened bit about `Slate`.

# Prerequisites

It is recommended to use [Homebrew](https://brew.sh) for package management. That only applies to macOS users.
You can install Homebrew globally using `brew`.

> Next, we need to use Brew to install Ruby:

Reviewer: ```suggestion
This project relies on `Ruby`. For macOS users, it is recommended to use [Homebrew](https://brew.sh) to install it:
```

`brew` and `Homebrew` refer to the same thing.

# Documentation

> This is the Ruby version of the documentation.

Reviewer: I think it would be good to expand on this a bit to mention that the project is based on Slate and add a link to the repo (https://github.com/slatedocs/slate).

# Prerequisites

> This project uses [Brew](https://brew.sh/index_fr) for package management. You can install Brew globally using `brew`:

Reviewer: This isn't necessarily true. `Homebrew` is just one potential way of installing the project dependencies and it isn't available on all platforms. It would be more accurate to state here that `Brew` is the recommended installation method and that it only applies to users of macOS. In addition, I think it would be better to link to the English language version of the `Homebrew` website (https://brew.sh) to maintain consistency with this README.

> ```shell
> $ /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
> ```

Reviewer: ```suggestion
```

I think it would be best not to include the installation instructions for `Homebrew` here because they could potentially change, which would make these instructions invalid. You've already linked to the `Homebrew` website in the above step, and anybody following these instructions can go to there to find the installation steps, which will always be accurate.

Next, we need to use Brew to install Ruby :

```shell
$ brew install ruby
```

> Next, we need to use [Bundler](https://bundler.io/) to install gems :

Reviewer: ```suggestion
Next, we need to use [RubyGems](https://rubygems.org) to install [Bundler](https://bundler.io):
```

The following step isn't using `Bundler` to install gems, it's using `RubyGems` to install `Bundler`.
